.jules/sentinel.md

```
@@ -0,0 +1,6 @@
+# Sentinel's Journal - Critical Security Learnings
+
+## 2026-03-08 - Sensitive Data Leakage in Logs and Console
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)